### PR TITLE
Implement HelloWorld class by using Java Spark

### PR DESCRIPTION
**Pull request by AndriiSameliuk**
This changeset configures the application with the REST endpoints demonstrating the Spark Java framework capabilities.

In particular, `HelloWorld` service is implemented. It accepts GET requests on `/hello` endpoint.

### CODE REVIEW

**Comment by AndriiSameliuk** [comment]
@BohdanDudkaTD, PTAL.

**Comment by AndriiSameliuk** [comment]
@okucheruk, @Oleh-Stefanovych, PTAL.

**Comment by AndriiSameliuk** [comment]
@armiol, PTAL.

**Comment by AndriiSameliuk** [comment]
@Oleg-Melnik, PTAL.

File: server/build.gradle [hunk 1]
```diff
@@ -17,4 +17,7 @@ dependencies {
     testCompile 'org.junit.jupiter:junit-jupiter-api:5.4.2'
     testCompile 'org.junit.jupiter:junit-jupiter-engine:5.4.2'
     testCompile 'org.junit.jupiter:junit-jupiter-params:5.4.2'
+    compile "com.sparkjava:spark-core:2.8.0"
+    compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.25'
```
Comment (armiol) [inline server/build.gradle]: Please extract the versions into a separate dictionary.

File: server/src/main/java/io/javaclasses/filestorage/HelloWorld.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+/**
+ * Entry point for HelloWorld application.
+ */
+public class HelloWorld {
+
+    /**
+     * Listens get request on /hello URL and shows a message on page.
+     * @param args command line arguments.
+     */
```
Comment (armiol) [inline server/src/main/java/io/javaclasses/filestorage/HelloWorld.java]:
```
Listens to {@code GET} requests on {@code /hello} endpoint.

<p> Displays a welcome message via the HTTP response.

@param args the command line arguments
```

File: server/src/main/resources/log4j.properties [hunk 3]
```diff
@@ -0,0 +1,13 @@
+log4j.rootLogger=DEBUG, stdout, file
```
Comment (armiol) [inline server/src/main/resources/log4j.properties]: Let's set the level to `INFO` and remove the `file` appender.
Also, please think of `deng`. Not sure that's what we really need.

File: server/src/main/java/io/javaclasses/filestorage/HelloWorld.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
+package io.javaclasses.filestorage;
```
Comment (armiol) [inline server/src/main/java/io/javaclasses/filestorage/HelloWorld.java]: We need `package-info.java`.

**Pull request merged**